api/src/main/java/net/countercraft/movecraft/craft/Craft.java

```diff
@@ -40,7 +40,9 @@
 import org.bukkit.block.data.BlockData;
 import org.bukkit.persistence.PersistentDataType;
 import org.jetbrains.annotations.NotNull;
+import org.joml.Vector3i;
 
+import javax.naming.Name;
```

Comment:
What are these new imports for?

Reply:
Good question, probably some left overs when i tried something else in my working copy, will remove them when i get home